libs/s3/s3_core_lib.py

```diff
@@ -64,7 +64,8 @@ def __init__(self,
             region = kwargs.get("region", None)
         aws_session_token = kwargs.get("aws_session_token", None)
         debug = kwargs.get("debug", S3_CFG["debug"])
-        config = Config(retries={'max_attempts': 6})
+        max_attempts = kwargs.get("max_attempts", 6)
```

Comment:
@DhobaleVishal As discussed, made changes to fetch max_attempts from kwargs. 
Default value is kept 6 as earlier.

Reply:
Remember this is boto3 specific.